cg/meta/workflow/balsamic.py

```diff
@@ -238,22 +237,29 @@ def get_verified_pon(self, panel_bed: str, pon_cnn: str) -> str | None:
                 raise BalsamicStartError(
                     f"The specified PON reference file {latest_pon} does not match the panel bed {panel_bed}"
                 )
+            latest_pon = self.get_verified_sex_pon(sample_sex=sex, pon_cnn=latest_pon)
+
         else:
-            latest_pon = self.get_latest_pon_file(panel_bed)
+            latest_pon = self.get_latest_pon_file(panel_bed=panel_bed, sex=sex)
             if latest_pon:
                 LOG.info(
                     f"The following PON reference file will be used for the analysis: {latest_pon}"
                 )
 
         return latest_pon
 
-    def get_latest_pon_file(self, panel_bed: str) -> str | None:
-        """Returns the latest PON cnn file associated to a specific capture bed"""
+    def get_latest_pon_file(self, panel_bed: str, sex: Sex) -> str | None:
```

Comment:
Could you add an additional pytest testing a sex specific PON?

Reply:
Yes! Just wanted to wait for the review first :D